crates/wdk-build/src/lib.rs

```diff
@@ -11,14 +11,15 @@
 //! models (WDM, KMDF, UMDF).
 
 #![cfg_attr(nightly_toolchain, feature(assert_matches))]
+use std::{fmt, str::FromStr};
 
 pub use bindgen::BuilderExt;
 use metadata::TryFromCargoMetadataError;
 
 pub mod cargo_make;
 pub mod metadata;
 
-mod utils;
+pub mod utils;
```

Comment:
This module is meant for private utils contained for use within this crate. It looks all that's used in `PathExt` and `StripExtendedPathPrefixError`. I think both those things are obsolete now that `https://doc.rust-lang.org/std/path/fn.absolute.html` is available on stable. I originally wrote that code before it was stabilized. 

All uses of `PathExt` because of canonicalize should be changed to use `path::absolute` and other stdlib functions. The `utils` module should also be returned to being private